src/pymodaq/utils/leco/daq_move_LECODirector.py

```diff
@@ -39,7 +40,7 @@ class DAQ_Move_LECODirector(LECODirector, DAQ_Move_base):
     controller: ActuatorDirector
 
     params_client = []  # parameters of a client grabber
-    data_actuator_type = DataActuatorType['float']  # DataActuatorType['DataActuator']
+    data_actuator_type = DataActuatorType['DataActuator']
```

Comment:
DataActuatorType.DataActuator would be better (I updated the template to reflect this)

Reply:
Done.